deploy/olm-catalog/bundle/manifests/konveyor-forklift-operator.v99.0.0.clusterserviceversion.yaml

```diff
@@ -121,6 +121,10 @@ spec:
                   value: forklift-controller
                 - name: CONTROLLER_TAG
                   value: latest
+                - name: VALIDATION_REPO
```

Comment:
We should probably add the validation image to the CSV related images for consistency around line 76.